H/hidapi/build_tarballs.jl

```diff
@@ -19,10 +19,13 @@ mkdir build
 cd build
 
 if [[ ${target} == *-linux-* ]]; then
-    # We need this only to trick `configure` into thinking that udev is available,
-    # but we aren't going to actually use it
     apk add eudev-dev
-    linuxflags="-DHIDAPI_WITH_HIDRAW=false"
+    linuxflags="-DHIDAPI_WITH_LIBUSB=false"
+    export CPATH=/usr/include
+fi
+
+if [[ ${target} == *-freebsd ]]; then
+    apk add libiconv
```

Comment:
What do you need of libiconv when targeting FreeBSD? Why not adding iconv_jll?

Reply:
Thats an oversight, didn't know that there is an iconv_jll.